connector/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatITTest.java

```diff
@@ -199,6 +221,7 @@ public void sinkEdgeData() throws ExecutionException, InterruptedException {
                         + PASSWORD
                         + "',"
                         + "'graph-space'='flinkSink',"
+                        + "'table-name'='friend',"
```

Comment:
so the `friendTable` is the view table name in flinksql, and the `friend` is the real edge name in NebulaGraph?
if so, is it better to use `label-name` but not `table-name`?

Reply:
agree